packages/cypher-ls/package.yaml

```
@@ -0,0 +1,16 @@
+---
+name: cypher-language-server
+description: Language Server for Cypher query language.
+homepage: https://github.com/neo4j/cypher-language-support
+licenses:
+  - Apache-2.0
+languages:
+  - Cypher
+categories:
+  - LSP
+
+source:
+  id: pkg:npm/@neo4j-cypher/language-server@2.0.0-next.1
```

The `@` sign will need to be [percent-encoded](https://en.wikipedia.org/wiki/Percent-encoding):
```suggestion
  id: pkg:npm/%40neo4j-cypher/language-server@2.0.0-next.1
```